velox/functions/prestosql/benchmarks/JsonFileReader.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <boost/algorithm/string/replace.hpp>
+#include <boost/filesystem.hpp>
+#include <google/protobuf/util/json_util.h>
+#include <fstream>
+#include <sstream>
+#include "velox/common/base/Exceptions.h"
+
+namespace fs = boost::filesystem;
+
+namespace {
+
+std::string getDataPath(const std::string& fileSize) {
+  std::string currentPath = fs::current_path().c_str();
+
+  // CLion runs the tests from cmake-build-release/ or cmake-build-debug/
+  // directory. Hard-coded json files are not copied there and test fails with
+  // file not found. Fixing the path so that we can trigger these tests from
+  // CLion.
+  boost::algorithm::replace_all(currentPath, "cmake-build-release/", "");
```

Comment:
FYI, If json files etc need to be present in same working directory as executables, then you need to specify them as a RESOURCE_FILE/BUNDLE in cmake(see https://cmake.org/cmake/help/latest/prop_tgt/RESOURCE.html) . That would be more appropriate solution I feel.

Reply:
Can we use the generation script to generate JSON files automatically as velox/functions/prestosql/json/tests/data/generate_json.py? Because the files are too large to be placed directly in the source code repository as lingbin has mentioned the #4053

Reply:
@kgpai,  Do you think generating JSON files from scripts is reasonable?  Or do we just need to test the small json files and upstream the files directly? Thanks!

Reply:
Hi, @kgpai ,  I have updated JsonExprBenchmark.cpp to generate json data in the prepareData() function instead of using json file to avoid big json files presented in the src directory. Do you think this is reasonable?

Reply:
Thanks @wanweiqiangintel , thats very reasonable. Thank you for this work. Going through PR again.